### PR TITLE
Add permission for users to edit their own topics

### PR DESCRIPTION
**Pull request by Aberdeener**
Removes confusion when users did not know they needed the "create replies" permission to edit their own topic.
*Can Moderate permission overrides this.*
I also added an updater file and updated the Queries to add this to the database upon installation.

### CODE REVIEW

**Comment by samerton** [comment]
Just to check, is this specifically for editing topics and not replies?

**Comment by Aberdeener** [comment]
Correct!

File: modules/Forum/pages/panel/forums.php [hunk 1]
```diff
@@ -753,6 +759,7 @@
 									'forum_id' => $_GET['forum'],
 									'view' => $view,
 									'create_topic' => $create,
+									'edit_topic' => $create,
```
Comment (samerton) [inline modules/Forum/pages/panel/forums.php:762]: This line should be `'edit_topic' => $edit,`

Comment (Aberdeener) [reply modules/Forum/pages/panel/forums.php]: Fixed

**Pull request merged**